index.html

```diff
@@ -11,35 +11,40 @@
   <body>
     <header>
       <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
+      <button type="button" id="open-menu-mobile" onclick="openMenuMobile()">
         <i class="fas fa-bars"></i>
       </button>
+
+      <nav id="mobile-nav">
+        <button type="button" id="close-menu-mobile" onclick="closeMenuMobile()">
+          <i class="fas fa-times"></i>
+        </button>
+        <ul class="mobile-nav-link-list">
+          <li><a class="mobile-nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="mobile-nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="mobile-nav-link" href="#contact">
+              Contact <i class="far fa-envelope"></i>
+            </a>
+          </li>
+        </ul>
+      </nav>
+
       <nav id="desktop-nav">
         <ul>
           <li><a class="nav-link" href="#projects">Portoflio</a></li>
           <li><a class="nav-link" href="#about-me">About</a></li>
           <li>
-            <a class="nav-link" href="#contact"
-              >Contact <i class="far fa-envelope"></i
-            ></a>
+            <a class="nav-link" href="#contact">
+              Contact <i class="far fa-envelope"></i>
```

Comment:
- [ ] As the previous reviewer already mentioned, there is an envelop icon next to 'Contact' when the menu is expanded. According to the template, it should not be visible in mobile view. It will make the application look more professional if we remove the envelope from the mobile view.